<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemAdd</class>
 <widget class="QWidget" name="ItemAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>285</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>354</width>
    <height>285</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>354</width>
    <height>285</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Наименование / Добавить</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="itemNameL">
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="itemName">
       <property name="minimumSize">
        <size>
         <width>241</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>241</width>
         <height>20</height>
        </size>
       </property>
       <property name="maxLength">
        <number>200</number>
       </property>
       <property name="placeholderText">
        <string>Введите название товара...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="itemNumberL">
       <property name="text">
        <string>Номер:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="itemNumber">
       <property name="minimumSize">
        <size>
         <width>165</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>165</width>
         <height>20</height>
        </size>
       </property>
       <property name="maxLength">
        <number>150</number>
       </property>
       <property name="placeholderText">
        <string>Введите уникальный номер...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Стоимость за сутки: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>руб.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>8-14 дней:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>руб.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>руб.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>руб.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>3-7 дней:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>руб.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>15-30 дней:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>1-2 дня:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>31 и более:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="itemPrice4">
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>20</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="itemPrice3">
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>20</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="itemPrice1">
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>20</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="itemPrice2">
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>20</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="itemPrice5">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="itemAddOkB">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="itemAddCancelB">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>itemName</tabstop>
  <tabstop>itemNumber</tabstop>
  <tabstop>itemPrice1</tabstop>
  <tabstop>itemPrice2</tabstop>
  <tabstop>itemPrice3</tabstop>
  <tabstop>itemPrice4</tabstop>
  <tabstop>itemPrice5</tabstop>
  <tabstop>itemAddOkB</tabstop>
  <tabstop>itemAddCancelB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
